doc/man7/ossl-guide-quic-multi-stream.pod

```diff
@@ -0,0 +1,379 @@
+passes the connection B<SSL> object as a parameter.
+
+If a client application calls L<SSL_write_ex(3)> or L<SSL_write(3)> first then
+(by default) the default stream will be a client-initiated bi-directional
+stream. If the client applications call L<SSL_read_ex(3)> or L<SSL_read(3)>
+first then the first stream initiated by the server will be used as the default
+stream (whether it is bi-directional or uni-directional).
+
+This behaviour can be controlled via the default stream mode. See
+L<SSL_set_default_stream_mode(3)> for further details.
+
+It is recommended that new multi-stream applications should not use a default
+stream at all and instead should use a spearate stream B<SSL> object for each
+stream that is used. This requires calling L<SSL_set_default_stream_mode(3)>
+and setting the mode to B<SSL_DEFAULT_STREAM_MODE_NONE>.
+
+=head1 CREATING NEW STREAMS
+
+An endpoint can create a new stream by calling L<SSL_new_stream(3)>. This
+creates a locally initiated stream. In order to do so you must pass the QUIC
+connection B<SSL> object as a parameter. You can also specify whether you want a
+bi-directional or a uni-directional stream.
+
+The function returns a new QUIC stream B<SSL> object for sending and receiving
+data on that stream.
+
+The peer may also initiate streams. An application can use the function
+L<SSL_get_accept_stream_queue_len(3)> to determine the number of streams that
+the peer has initiated that are waiting for the application to handle. An
+application can call L<SSL_accept_stream(3)> to create a new B<SSL> object for
+a remotely initiated stream. If the peer has not initiated any then this call
+will block until one is available if the connection object is in blocking mode
+(see L<SSL_set_blocking_mode(3)>).
+
+When using a default stream OpenSSL will prevent new streams from being
+accepted. To override this behaviour you must call
+L<SSL_set_incoming_stream_policy(3)> to set the policy to
+B<SSL_INCOMING_STREAM_POLICY_ACCEPT>. See the man page for further details. This
+is not relevant if the default stream has been disabed as described in
+L</THE DEFAULT STREAM> above.
+
+Any stream may be bi-directional or uni-directional. If it is uni-directional
+then the initiator can write to it but not read from it, and vice-versa for the
+peer. You can determine what type of stream an B<SSL> object represents by
+calling L<SSL_get_stream_type(3)>. See the man page for further details.
+
+=head1 USING A STREAM TO SEND AND RECEIVE DATA
+
+Once you have a stream B<SSL> object (which includes the connection B<SSL>
+object if a default stream is in use) then you can send and receive data over it
+using the L<SSL_write_ex(3)>, L<SSL_write(3)>, L<SSL_read_ex(3)> or
+L<SSL_read(3)> functions. See the man pages for further details.
+
+In the event of one of these functions not returning a success code then
+you should call L<SSL_get_error(3)> to find out further details about the error.
```

Comment:
Consider removing `then` in this sentence

Reply:
It makes sense to me to keep it?